x-pack/plugin/sql/qa/server/src/main/resources/docs/docs.csv-spec

```diff
@@ -676,9 +676,9 @@ SELECT gender AS g, ROUND((MIN(salary) / 100)) AS salary FROM emp GROUP BY gende
 
        g       |    salary     
 ---------------+---------------
-null           |253            
-F              |260            
-M              |259        
+null           |253
+F              |259
```

Comment:
How come this changed?

Reply:
`MIN(salary)` == 25976. Before the change, this is returned as float, so it's rounded up. After the change, the `/100` division is done on integers, which rounds the result down.

Reply:
Great, thx!